src/inc/quic_datapath.h

```diff
@@ -74,10 +95,11 @@ typedef enum CXPLAT_ECN_TYPE {
 inline
 uint16_t
 MaxUdpPayloadSizeFromMTU(
```

Comment:
Is there a way we can achieve this without this change? i.e. Fake UDP MTU, but reduce the max MTU instead to account for the difference?

Reply:
Update `CxPlatSocketGetLocalMtu` when using TCP?

Reply:
Or, maybe just don't do anything special, and let PMTUD handle it...

Reply:
> Or, maybe just don't do anything special, and let PMTUD handle it...
Wouldn't this overrun the packet allocated by CxPlatSendDataAlloc?

> Update CxPlatSocketGetLocalMtu when using TCP?
Seems like a good idea.